rfcs/text/0042/risk.yml

```diff
@@ -0,0 +1,144 @@
+---
+- name: risk
+  title: Risk information
+  group: 2
+  short: Fields for describing risk score and level.
+  beta: >
+    These fields are in beta and are subject to change.
+  description: >
+    Fields for describing risk score and risk level of entities such as hosts and users.
+    These fields are not allowed to be nested under `event.*`. Please continue to use
+    `event.risk_score` and `event.risk_score_norm` for event risk.
+  reusable:
+    top_level: false
+    expected:
+      - host
+      - user
+  type: group
+  fields:
+    - name: identifier_field
+      level: extended
+      type: keyword
+      example: host.name
+      description: >
+        Identifies which field was used to group/calculate this risk score.
+        Typically, hosts use `host.name` and users use `user.name`.
+    - name: identifier_value
+      level: extended
+      type: keyword
+      example: host-001.local
+      description: >
+        Identifies which field value was used to group/calculate this risk score.
+        Typically, hosts use values from `host.name` and users use values from `user.name`.
+    - name: category_1_score
+      level: extended
+      type: float
+      example: 33.0
+      description: >
+        The contribution of Category 1 to the overall risk score (`calculated_score`). Category 1 contains Detection Engine Alerts.
```

Comment:
@SourinPaul please review these descriptions, I feel like they could use some improvement 😄 .

Reply:
@rylnd reads great. 

A few suggestions. You can ignore any you do not like.

- identifier_field: An ECS field that identifies an entity. Typically, hosts use the field `host.name`, and users use the field `user.name`.
- identifier_value: The entity identifier used to calculate risk. Typically, hosts use values from `host.name` and users use values from `user.name`.
- category_x_score: The risk contribution from a risk category (e.g., Category 'x') to the overall entity risk score. 
Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others. 
- category_x_count: The number of risk input documents that contributed to each category score, in this case, category '1'. 
Risk Categories logically group risk inputs from various domain use cases - e.g. Alert, Posture Management, Vulnerabilities, and others.

Reply:
@SourinPaul I added your feedback in [a6df8ea](https://github.com/elastic/ecs/pull/2236/commits/a6df8ea853bb53279066e54a465d18019b99d5d7). However, because we can't describe the fields generally and instead need to define them individually, I left the previous copy about what the particular category includes (for the `_score`s, at least). If you don't want to be that specific and would rather remove that aspect, please let me know.

Reply:
I agree that individual definitions are needed and your correct definitions are adequate. Thanks @rylnd!